zschema/compounds.py

```diff
@@ -209,6 +224,27 @@ def to_bigquery(self, name):
         }
         return retv
 
+    def to_proto(self, name, indent):
+        anon = False
+        try:
+            message_type = _proto_message_name(self.type_name)
+        except: # anonymous message
+            message_type = _proto_message_name(self.key_to_proto(name)) + "Struct"
+            anon = True
+
+        global _proto_messages
+        if anon or message_type not in _proto_messages:
+            fields = [v.to_proto(k, indent) for k,v in \
+                      sorted(self.definition.iteritems(), key=lambda (k,v): v.sort_index)]
+            fields = ["%s = %d;" % (fields[i], i+1) for i in range(len(fields))]
+            proto_def = "message %s {\n%s\n}" % (message_type, _proto_indent("\n".join(fields), indent+1))
+            _proto_messages[message_type] = True
+            if not anon:
+                print proto_def
```

Comment:
Is this ended to be a print here?

Reply:
Yes.  It's emitting a message definition at the global level, rather than an embedded anonymous message definition.